packages/appChrome/components/HeaderBar.tsx

```diff
@@ -1,15 +1,68 @@
 import * as React from "react";
 import { cx } from "emotion";
-import { darkMode, flex, padding } from "../../shared/styles/styleUtils";
+import {
+  background,
+  darkMode,
+  flex,
+  padding
+} from "../../shared/styles/styleUtils";
 import { headerBar, appChromeInsetContent } from "../style";
 
 export interface HeaderProps {
+  backgroundColor?: string;
   children: React.ReactElement<HTMLElement> | string;
+  paddingBottom?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingLeft?:
```

Comment:
The design team always wants the left and right padding to be the same value, and the top and bottom padding to be the same value.

So we could use the value from a prop like `paddingHoriz="16px"` to set 16px left _and_ right padding styles in the component, and `paddingVert="16px"` to set 16px top _and_ bottom padding styles in the component.

Reply:
Great suggestion.